csrankings-r.csv

```diff
@@ -957,6 +957,7 @@ Roberto Baldoni,Sapienza University of Rome,http://www.diag.uniroma1.it/users/ro
 Roberto Barbuti,University of Pisa,http://pages.di.unipi.it/barbuti,NOSCHOLARPAGE
 Roberto Battiti,University of Trento,https://rtm.science.unitn.it/~battiti,fO-wkZkAAAAJ
 Roberto Beraldi,Sapienza University of Rome,http://www.diag.uniroma1.it/users/roberto%20beraldi,WaJQ4-wAAAAJ
+Roberto Bondesan,Imperial College London,https://scholar.google.com/citations?user=l2z7p3oAAAAJ&hl=en,2z7p3oAAAAJ

```

Comment:
Google Scholar is not suitable as a home page. Please point to a personal or institutional web page. Also, Scholar says this faculty member is at Qualcomm - please clarify.

Reply:
Linked official public university staff page shows Roberto is a faculty member. However, after checking it seems he's joining very soon, but have not yet. Thus, no personal page exists. I've therefore deleted him from this pull request.